MKSOutputDevice.py

```diff
@@ -403,11 +404,27 @@ def get_max_filename_len(self):
                 return int(global_container_stack.getMetaDataEntry(Constants.MAX_FILENAME_LEN))
         return 30
 
+    def is_auto_file_renaming_enabled(self):
+        global_container_stack = Application.getInstance().getGlobalContainerStack()
+        if global_container_stack:
+            meta_data = global_container_stack.getMetaData()
+            if Constants.AUTO_FILE_RENAMING in meta_data:
+                return bool(global_container_stack.getMetaDataEntry(Constants.AUTO_FILE_RENAMING))
```

Comment:
Looks like we can just return **True**, because we don't save a **false** value (_see the 
"global_container_stack.removeMetaDataEntry(Constants.AUTO_FILE_RENAMING)" line_)

Reply:
Agree, updated